### DISCUSSION TITLE
[Question/future request] /helpop

### DISCUSSION THREAD

**Discussion by WolverStones** [Zrips/CMI#1476] [Enhancement]
It is possible to enable bungee support for the /helpop command such as /staffmsg command ... ?

Thanks...

**Comment by BlindMag** [comment]
Was this ever implemented?

**Comment by WolverStones** [comment]
> Was this ever implemented?



not yet, cmi has a limited bungee function

**Comment by pijlwagen** [comment]
bumping this because it's a good suggestion
